src/ralph/api/routers/statements.py

```diff
@@ -35,6 +35,17 @@
     settings.BACKENDS.DATABASE, settings.RUNSERVER_BACKEND.upper()
 ).get_instance()
 
+POST_PUT_RESPONSES = {
+    400: {
+        "model": ErrorDetail,
+        "description": "The request was invalid.",
+    },
+    409: {
+        "model": ErrorDetail,
+        "description": "Statements had a conflict with existing statements.",
+    },
+}
+
 
 @router.get("/")
```

Comment:
Given your improvements should we also accept `/xapi/statements` endpoint for GET requests?

```suggestion
@router.get("/")
@router.get("")
```

Reply:
From me reading it seemed like an empty string route doesn't work, so I left it as is. I will experiment and see what happens.

Reply:
I actually have no idea why I thought that, I've made the update 😓